src/test/java/org/prebid/server/handler/StatusHandlerTest.java

```diff
@@ -78,4 +82,23 @@ public void shouldRespondWithNoContentWhenMessageWasNotDefined() {
         // then
         verify(httpResponse).setStatusCode(eq(204));
     }
+
+    @Test
+    public void shouldRespondWithContentTypeHeaders() throws JsonProcessingException {
+        // given
+        statusHandler = new StatusHandler(Arrays.asList(healthCheck), jacksonMapper);
+
+        given(healthCheck.name()).willReturn("healthCheckName");
+        given(healthCheck.status()).willReturn(StatusResponse.of("healthCheckStatus", null));
+
+        given(routingContext.response()).willReturn(httpResponse);
+        given(httpResponse.putHeader(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON))
```

Comment:
This is not required since the last point of test is to verify `putHeader` method.